x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -64,6 +66,10 @@ export const getElasticsearchMetricQuery = (
   filterQuery?: string
 ) => {
   const interval = `${timeSize}${timeUnit}`;
+  const to = Date.now();
+  const intervalAsSeconds = getIntervalInSeconds(interval);
+  const from = to - intervalAsSeconds * 5000;
```

Comment:
Why * `5000`?

Reply:
That will give us 5 buckets and convert it to milliseconds.

Reply:
Makes sense but maybe add a comment or named variable explaining as such?